src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle," + " area: " + calculateArea() + " sq. units, "

```

Comment:
Magic numbers should be constants. It's a good practice to define 'PI' as a constant if you are going to use it for calculations. Although Math.PI is a constant, if you need a specific precision or for the sake of consistency with other figures' calculations, you might want to define it within your class.